Componets/Badge/badge.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Documentation|Badge</title>
+    <link rel="stylesheet" href="/style.css">
+    <script src="https://kit.fontawesome.com/52c15eb80a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <!--------- header---------->
+    <header class="header-container">
+        <a href="#" class="nav-link header-title"><h2>Matrix <span>UI</span></h2></a>
+       
+        <ul class="nav-list nav-items">
+            <li class="nav-item ">
+                <a class="nav-link " href="#">Home</a> </li>
+            <li class="nav-item">
+                <a class="nav-link active" href="#">Documentation</a></li>
+
+        </ul>
+        <ul class="nav-list nav-social-media">
+            <li class="nav-item">
+                <a class="nav-link" href="https://twitter.com/__sanjay13"><i class="fab fa-twitter"></i></a> </li>
+            <li class="nav-item">
+                <a class="nav-link" href="https://github.com/SanjayJatti"><i class="fab fa-github"></i></a></li>
+            <li class="nav-item">
+                <a class="nav-link" href="https://www.linkedin.com/in/sanjay-jatti-09a176140/"><i class="fab fa-linkedin"></i></a></li>
+        </ul> 
+    </header>
+      <!--------- sidebar---------->
+      <div class="sidebar">
+        <ul class = "sidebar-list">
+            <a href = "" class = "sidebar-item">Avatar</a>
+            <a href = "" class = "sidebar-item">Alert</a>
+            <a href = "" class = "sidebar-item sidebar-active">Badges</a>
+            <a href = "" class = "sidebar-item">Buttons</a>
+            <a href = "" class = "sidebar-item">Cards</a>
+            <a href = "" class = "sidebar-item">Responsive Images</a>
+            <a href = "" class = "sidebar-item">Forms</a>
+            <a href = "" class = "sidebar-item">Modal</a>
+            <a href = "" class = "sidebar-item">Navbar</a>
+            <a href = "" class = "sidebar-item">List</a>
+            <a href = "" class = "sidebar-item">Responsive Grid</a>
+            <a href = "" class = "sidebar-item">Typography</a>
+        </ul>
+    </div>
+      <!--------- Badges---------->
+    <div class="page-wrapper">
+        <h2>Badge</h2>
+        <p class="vertical-gutter-md">Badges are small circles, positioned either at top-right or bottom-right of the parent component. Badge can be used to display numbers, online / offline status, depending on where they are used. <br/>Copy the styles of different badges and use it in your code</p>
```

Comment:
I liked how you have used utility classes here to maintain consistent spacing throughout the project.